src/components/App.tsx

```diff
@@ -1,15 +1,18 @@
 import React from "react";
+import { useState } from "react";
 import { LoginForm } from "./LoginForm";
+import { MessageTab } from "./MessageTab";
 
 export const App: React.FC = () => {
   const [isLoggedIn, setIsLoggedIn] = React.useState(false);
+  const [unreadCount, setUnreadCount] = useState(0);
 
   return (
     <div className="w-96 h-96 bg-white p-4">
       {!isLoggedIn ? (
         <LoginForm onLogin={() => setIsLoggedIn(true)} />
       ) : (
-        <div>Chat interface will go here</div>
+        <MessageTab unreadCount={unreadCount} setUnreadCount={setUnreadCount} />
```

Comment:
Don't think we'd need to pass `setUnreadCount` down to the component. It's likely that we'd get the number of unread count from an API call in this component, or further up from `MessageTab`. That being said, it's not a blocker for this PR.